frontend/src/metabase/admin/settings/selectors.js

```diff
@@ -149,6 +150,9 @@ const SECTIONS = [
         ]
     }
 ];
+for (const section of SECTIONS) {
```

Comment:
The functional nazi in me wants to suggest declaring a separate `RAW_SECTIONS` constant and then map & i.assoc on that to build `SECTIONS`, but feel free to ignore him. =P